src/test/scala/org/apache/spark/sql/delta/DeltaGenerateSymlinkManifestSuite.scala

```diff
@@ -114,6 +123,15 @@ trait DeltaGenerateSymlinkManifestSuiteBase extends QueryTest
       generateSymlinkManifest(tablePath.toString)
       assertManifest(tablePath, expectSameFiles = true, expectedNumFiles = 0)
       assert(spark.read.format("delta").load(tablePath.toString).count() == 0)
+
+      // delete all data
+      write(5, 5, 5)
+      assertManifest(tablePath, expectSameFiles = false, expectedNumFiles = 0)
```

Comment:
This is a slightly incorrect test. There are already no manifest files before deleting the table. So its faulty to check whether deletion + generation generated zero files if there are already zero files.

I can fix this while merging your PR. I think all we need is to do is

```
write(5, 5, 5)
generateSymlinkManifest(tablePath.toString)
assertManifest(tablePath, expectSameFiles = true, expectedNumFiles = 25)
```